src/cdk/v2/destinations/zoho/utils.js

```diff
@@ -11,13 +11,13 @@ const { isHttpStatusSuccess } = require('../../../../v0/util');
 const { handleHttpRequest } = require('../../../../adapters/network');
 const { CommonUtils } = require('../../../../util/common');
 
-const deduceModuleInfoV2 = (Config, destConfig) => {
+const deduceModuleInfoV2 = (Config, destConfig, deliveryAccount) => {
   const { object, identifierMappings } = destConfig;
   const identifierType = identifierMappings.map(({ to }) => to);
   return {
     operationModuleType: object,
     upsertEndPoint: ZOHO_SDK.ZOHO.getBaseRecordUrl({
-      dataCenter: Config.region,
+      dataCenter: deliveryAccount?.options?.region || Config.region,
```

Comment:
I am assuming config and deliveryAccount present in destination so can we  write util function to abstract this fallback logic.
function should something like this getRegion(destination)

Reply:
Added one function for this.